src/commands/workers/deploy.ts

```diff
@@ -64,8 +65,8 @@ export default class Deploy extends BaseCommand<typeof Deploy> {
       true
     );
 
-    const fluenceClient = await initFluenceClient(flags, fluenceConfig);
-    doRegisterIpfsClient(fluenceClient, flags["off-aqua-logs"]);
+    await initFluenceClient(flags, fluenceConfig);
+    doRegisterIpfsClient(flags["off-aqua-logs"]);
```

Comment:
It's kinda strange that `ipfs client` receives flag `off-aqua-logs`. What is happening here? :)

Reply:
Ah, I got it. `IpfsClient` is a service, and there's a `log` function in there. 

So some scripts do `IpfsClient.log`? That's pretty strange still. Maybe no scripts do that, and we can remove it safely?

Reply:
What do you wanna remove? logs in ipfs client use commandObj.log. Yeah a reused off-aqua-logs for that as well so if this flag is used there would be no such logs printed as a result of aqua execution

Reply:
I guess just renaming to `off-ipfs-client-logs` would make sense

Reply:
Discussed in DM to turn them off for now completely. Done